pkgs/applications/misc/apvlv/default.nix

```diff
@@ -58,7 +58,7 @@ stdenv.mkDerivation rec {
     '';
 
     license = licenses.lgpl2;
-    platforms = platforms.unix;
+    platforms = platforms.linux;
```

Comment:
hexdump wasn't enough to get this working?

Reply:
Nope, it breaks on "gdk/gdkx.h" file not found. Looks like it's since been fixed but no release has come out: 68482526dd9c977c4f9777fd27c99a8d73609cc0

Reply:
hmm, maybe I'm confusing it with another package then. 😄